application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -45,6 +46,9 @@
     private static final String MAX_USES_OPTION = "max-uses";
     private static final String MAX_AGE_OPTION = "max-age";
 
+    private static final long MAX_VALUE_MAX_AGE = TimeUnit.SECONDS.toDays(604800);
```

Comment:
```suggestion
    private static final long MAX_VALUE_MAX_AGE = TimeUnit.DAYS.toSeconds(7);
```
604800 is still unreadable, I'm too lazy to grab my calculator to find out how many days that is.
We wanted to use the TimeUnit exactly so the 604800 could get removed.